src/reader/ns_reader.rs

```diff
@@ -48,6 +50,12 @@ impl<R> NsReader<R> {
     pub fn config_mut(&mut self) -> &mut Config {
         self.reader.config_mut()
     }
+
+    /// Returns all the prefixes currently declared except the default xml: and xmlns: namespaces.
```

Comment:
I think, we need also adopt tests from `tests/namespace.rs` to include them as doctests here to show the behavior. I suppose to show results for this XML:
```xml
<root>
  <a xmlns="a1" xmlns:a="a2">
    <b xmlns="b1" xmlns:b="b2">
      <c/>
    </b>
    <d/>
  </a>
</root>
```

Reply:
Done!